teams/teams-ps/teams/Get-CsTeamsAIPolicy.md

```diff
@@ -0,0 +1,54 @@
+---
+external help file: Microsoft.TeamsCmdlets.PowerShell.Custom.dll-Help.xml
+Module Name: MicrosoftTeams
+applicable: Microsoft Teams
+title: Get-CsTeamsAIPolicy
+online version: https://learn.microsoft.com/powershell/module/teams/Get-CsTeamsAIPolicy
+schema: 2.0.0
+author: Andy447
+ms.author: andywang
+---
+
+# Get-CsTeamsAIPolicy
+
+## SYNOPSIS
+
+This cmdlet retrieves all Teams AI policies for current tenant.
+
+## SYNTAX
+
+```
+Get-CsTeamsAIPolicy
+```
+
+## DESCRIPTION
+
+This cmdlet retrieves all Teams AI policies for current tenant.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Get-CsTeamsAIPolicy
+```
+
+retrieves Teams AI policies, shows "EnrollFace" and "EnrollVoice" values.
```

Comment:
Please capitalize the first letter of the word "Retrieves". To confirm, this cmdlet retrieves all Teams AI policies created for this tenant and shows what the EnrollFace and EnrollVoice values are for each policy?